docs/versioning/android-version.md

```diff
@@ -16,8 +16,8 @@ In order to manage version code and version name with Appcircle, two requirement
 - The build's Version Management toggle must be turned on and required input values must be entered
 - **Increment Build and Version Number** component 1.0.* or higher must be in your workflow
 
-<NarrowImage src="https://cdn.appcircle.io/docs/assets/android-versioning1.png" />
-<NarrowImage src="https://cdn.appcircle.io/docs/assets/android-versioning2.png" />
+<Screenshot url="https://cdn.appcircle.io/docs/assets/versioning-android-configuration.png" />
+<Screenshot url="https://cdn.appcircle.io/docs/assets/versioning-android-workflow.png" />
```

Comment:
According to our latest guidelines there should be description betwwen images.

Here both images have their seperate description. I can improve them... :wrench: